### ISSUE TITLE
Add a command to launch sbt shell in the builtin terminal

### ISSUE THREAD

**Issue by laughedelic** [laughedelic/atom-ide-scala#66]
Since v0.11 Atom IDE UI has a terminal and it seems to be possible to open it with a given command: https://github.com/facebook-atom/atom-ide-ui/issues/220#issuecomment-383728210.

It would be nice to have a command to open sbt shell inside Atom. It could be also useful to suggest it (as a notification with a button) when launching Metals, if there is no running sbt server yet (`project/target/active.json`).

**Comment by cquiroz** [comment]
I think this would be very useful

**Comment by laughedelic** [comment]
@cquiroz let me know if you want to work on this. 

Adding a command should be easy, but opening the terminal with the right configuration needs some investigation (see the linked comment). I'm not 100% it's currently possible, but if not, it's a valid feature request to atom-ide-ui.

**Comment by cquiroz** [comment]
I supposed I could try though I'm quite short on time.

**Comment by laughedelic** [comment]
No problem. There's no time limit. I don't think I'm going to dig into this any time soon, so it'll stay an open challenge.

**Comment by cquiroz** [comment]
Am i correct that the first thing is to add a facade to `scalajs-atom-api` for `atom-ide-terminal`?

**Comment by laughedelic** [comment]
Yes, that would be very helpful. But you don't need to write facades for everything right away. Try to experiment with the available API first (with `js.Dynamic`) and see what's needed.
